server/src/test/java/com/objectcomputing/checkins/services/team/TeamServicesImplTest.java

```diff
@@ -29,9 +27,6 @@
     @Mock
     private TeamMemberRepository teamMemberRepository;
 
-    @Mock
-    private MemberProfileRepository memberProfileRepository;
-
     @InjectMocks
```

Comment:
Do we need this test class? Since we are doing end-to-end tests from teamControllerTest.java, these tests are retesting the tested code?